redaxo/src/addons/install/lang/en_gb.lang

```diff
@@ -43,13 +43,15 @@ install_info_core_updated = Core has been updated!
 install_warning_core_not_updated = The core could not be updated for the following reason:
 install_warning_core_zip_not_extracted = The zip file could not be unpacked in the "src" directory!
 
-install_info_addon_downloaded = AddOn "{0}" downloaded!
-install_warning_addon_not_downloaded = AddOn "{0}" could not be downloaded due to the following reasons:
-install_info_addon_updated = AddOn "{0}" has been updated!
-install_warning_addon_not_updated = AddOn "{0}" could not be downloaded due to the following reasons:
-install_info_addon_uploaded = AddOn "{0}" has been uploaded!
-install_info_addon_deleted = AddOn-Version of "{0}" has been deleted!
+install_info_addon_downloaded = AddOn <b>{0}</b> downloaded!
+install_warning_addon_not_downloaded = AddOn <b>{0}</b> could not be downloaded due to the following reasons:
+install_info_addon_updated = AddOn <b>{0}</b> has been updated!
+install_warning_addon_not_updated = AddOn <b>{0}</b> could not be downloaded due to the following reasons:
+install_info_addon_uploaded = AddOn <b>{0}</b> has been uploaded!
+install_info_addon_deleted = AddOn-Version of <b>{0}</b> has been deleted!
 install_warning_addon_zip_not_extracted = Zip file could not bee extracted in the AddOn directory. You may have missing write permissions or the folder name within the zip file does not match the AddOn-Key.
+install_warning_message_from_addon = AddOn <b>{0}</b> reports:
+install_warning_message_from_plugin = PlugIn <b>{0}</b> reports:
```

Comment:
Passt das so?
(War mir nicht sicher, wenn ich es in Traduko gebe, ob der Kontext klar ist..)